cluster/aws/vivid/util.sh

```diff
@@ -34,6 +34,10 @@ function detect-vivid-image () {
         AWS_IMAGE=ami-ee023e80
         ;;
 
+      ap-northeast-2)
+        AWS_IMAGE=ami-c2df11ac
```

Comment:
For the record, I verified this ID against https://cloud-images.ubuntu.com/locator/